imagersite/imagersite/templates/registration/activation_complete.html

```diff
@@ -0,0 +1,6 @@
+{% extends "imagersite/base.html" %}
```

Comment:
You should add link to the home page or login here so that when a user registered they have somewhere to go.